### PR TITLE
MINOR: Increase produce timeout to 120 seconds

### PR DESCRIPTION
**Pull request by wicknicks**
This gives more room to pass this test on systems with low
resources running many parallel tests.

Signed-off-by: Arjun Satish <arjun@confluent.io>

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedKafkaCluster.java [hunk 1]
```diff
@@ -75,7 +75,7 @@
 
     private static final Logger log = LoggerFactory.getLogger(EmbeddedKafkaCluster.class);
 
-    private static final long DEFAULT_PRODUCE_SEND_DURATION_MS = 500;
+    private static final long DEFAULT_PRODUCE_SEND_DURATION_MS = TimeUnit.MILLISECONDS.convert(120, TimeUnit.SECONDS);
```
Comment (rhauch) [inline connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedKafkaCluster.java]: Wouldn't this be a bit cleaner?
```suggestion
    private static final long DEFAULT_PRODUCE_SEND_DURATION_MS = TimeUnit.SECONDS.toMillis(120); 
```

File: connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedKafkaCluster.java [hunk 2]
```diff
@@ -254,7 +254,7 @@ public void produce(String topic, Integer partition, String key, String value) {
         try {
             producer.send(msg).get(DEFAULT_PRODUCE_SEND_DURATION_MS, TimeUnit.MILLISECONDS);
         } catch (Exception e) {
-            throw new KafkaException("Could not produce message to topic=" + topic, e);
+            throw new KafkaException("Could not produce message: " + msg, e);
```
Comment (rhauch) [inline connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedKafkaCluster.java:257]: Can we still include the topic name in the exception?

**Review by rhauch — COMMENTED** [review]
Thanks, @wicknicks! Two minor suggestions below.

File: connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedKafkaCluster.java (hunk 2 above)
Comment (wicknicks) [reply connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedKafkaCluster.java:257]: the topic name is included in the msg (that's why I removed it here).

Comment (rhauch) [reply connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedKafkaCluster.java:257]: Ah, nevermind, the topic is in the message's `toString()`.

**Comment by rhauch** [comment]
The one JDK 8 test failure is unrelated.

**Review by rhauch — APPROVED** [review]
LGTM. Thanks, @wicknicks.

**Comment by rhauch** [comment]
Merged back to the `2.0` branch.

**Comment by mjsax** [comment]
@wicknicks @rhauch I think this PR was for KAFKA-7999? If yes, please link the PR and update the ticket. Thanks.

**Comment by wicknicks** [comment]
@mjsax ticket was resolved.

**Pull request merged**